conda/cli/find_commands.py

```diff
@@ -61,16 +61,20 @@ def find_commands(include_others=True):
     dir_paths.extend(os.environ.get("PATH", "").split(os.pathsep))
 
     if on_win:
-        pat = re.compile(r"conda-([\w\-]+)\.(exe|bat)$")
+        pat = re.compile(r"conda-([\w\-]+)(\.(exe|bat))?$")
     else:
         pat = re.compile(r"conda-([\w\-]+)$")
 
     res = set()
     for dir_path in dir_paths:
         if not isdir(dir_path):
```

Comment:
We could even move the try/except up here and catch not a directory error + permission error (or OSError?)

Reply:
see https://github.com/conda/conda/pull/13086/commits/59adb5c6901a722f03667a2e4353e285378b2a71